Xwt.XamMac/Xwt.Mac/LabelBackend.cs

```diff
@@ -96,6 +97,10 @@ public void SetFormattedText (FormattedText text)
 				Wrap = WrapMode.Character;
 			Widget.AllowsEditingTextAttributes = true;
 			Widget.AttributedStringValue = text.ToAttributedString ();
```

Comment:
Maybe we can avoid double drawing of the string if we add an extension in MacUtil.cs:

```
public static NSMutableAttributedString ToMutableAttributedString (this FormattedText ft)
{
    return (NSMutableAttributedString)ToAttributedString (ft);
}

And use this before calling SetAlignmentAttribute.

Then, `SetAlignmentAttribute` would become
```
void SetAlignmentAttribute (NSMutableAttributedString ns)
{
 			if (ns == null)
 				return;
 			
 			ns.BeginEditing ();
 			var r = new NSRange (0, ns.Length);
 			ns.RemoveAttribute (NSStringAttributeKey.ParagraphStyle, r);
 			var pstyle = NSParagraphStyle.DefaultParagraphStyle.MutableCopy () as NSMutableParagraphStyle;
 			pstyle.Alignment = Widget.Alignment;
 			ns.AddAttribute (NSStringAttributeKey.ParagraphStyle, pstyle, r);
 			ns.EndEditing ();
 			Widget.AttributedStringValue = ns;
 		}
```